plugins/generator-1.16.4/forge-1.16.4/procedures/explode.java.ftl

```diff
@@ -1,3 +1,3 @@
-if(world instanceof World && !world.isRemote()) {
+if(world instanceof World && !((World) world).isRemote) {
 	((World) world).createExplosion(null,(int)${input$x},(int)${input$y},(int)${input$z},(float)${input$power}, Explosion.Mode.BREAK);

```

Comment:
Mode is now always break in 1.16.4 so this PR will not work in 1.16.4 and default to break

Reply:
Fixed.